tests/unit/resolution_resolvelib/test_resolver.py

```diff
@@ -69,6 +69,9 @@ def test_new_resolver_get_installation_order(resolver, edges, ordered_reqs):
                 graph.add(v)
         graph.connect(parent, child)
 
+    # Sanity check: Make sure the vertice count is correct.
```

Comment:
Nit: `vertice` -> `vertex`

Reply:
Singular: vertex; plural: vertices. I’ll remember this for the rest of my life after implementing ResolveLib.

Reply:
Haha, BTW I am not sure if `vertices count` is better here, or `vertex count`.

Reply:
Vertex count is more natural.

I feel like I should apologise for the stupid grammar rules in English at this point...

Reply:
Dropped this commit in a recent force-push. :)